modules/sandboxed-containers-selecting-nodes.adoc

```diff
@@ -3,5 +3,31 @@
 // * sandboxed_containers/deploying_sandboxed_containers.adoc
 
 [id="sandboxed-containers-selecting-nodes_{context}"]
+= Selecting nodes for {sandboxed-containers-first}
 
-= Selecting nodes for OpenShift sandboxed containers
+You can selectively install the Kata runtime on specific workers.
+
+.Prerequisites
+
+* You have {product-title} 4.8 installed on your cluster.
+* You have installed the OpenShift CLI (oc).
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Identify the labels you want to use for selecting your nodes. For this example, select nodes with labels to be chosen as candidates to run on your {sandboxed-containers-first} workloads. If the nodes exist, they are selected.
+.. To apply a label to a node, run the following command: `oc label node <worker_node_name> custom-kata=test`. This labels your worker node with the `custom-kata` label that has a value of `test`.
+. Edit `KataConfig` custom resource (CR) `oc edit kataconfig` to add a label selector.
++
+[source,yaml]
+----
+  apiVersion: kataconfiguration.openshift.io/v1
+  kind: KataConfig
+  metadata:
+    name: cluster-kataconfig
+  spec:
+    kataConfigPoolSelector:
+      matchLabels:
+         custom-kata1: test
+----
+. Alternatively, you may use `oc apply -f <kata-config-with-label-selector.yaml>`.
```

Comment:
"Alternatively" - but these are three ordered steps. Are 2 & 3 two ways to do the same thing? If so, it might make sense to reorganize this procedure a little bit. Lmk if that's the case and I can elaborate what I'm thinking :)